src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/core/GenericContext.java

```diff
@@ -15,9 +15,34 @@
 
 package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.core;
 
-public abstract class GenericContext {
```

Comment:
let's keep this class as an abstract class. I don't think we need to move everything into this base class as we might have something completely different in SymptomContext.java

Reply:
Done